packages/storybook/src/DataVisualization/AudienceIcon.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import { SvgIcon } from '@material-ui/core'
```

Comment:
💡 should we use the one in Kodiak instead?

Reply:
oh good catch!  I used auto import and just assumed it pulled in the right one!

Reply:
I'll check if we can back out @material-ui/core because I think it was just used for storybook.

Reply:
hmm it's just used in table but will take a bit to pull it out so I'll leave it for now and keep an eye 👁️ on material-ui/core for SvgIcon coming in in the future.